packages/input-tel-dropdown/src/LionInputTelDropdown.js

```diff
@@ -281,13 +291,30 @@ export class LionInputTelDropdown extends LionInputTel {
     }
   }
 
+  reset() {
+    super.reset();
+    this._initSetModelValueBasedOnDropdown();
+  }
+
+  _initSetModelValueBasedOnDropdown() {
+    if (!(this.prefilled && this.dirty) && this._phoneUtil) {
+      if (!this._initialModelValue) {
+        const countryCode = this._phoneUtil.getCountryCodeForRegionCode(this.activeRegion);
+        this._initialModelValue = `+${countryCode}`;
+        this.modelValue = this._initialModelValue;
+      }
+      this.initInteractionState();
+      this.prefilled = false;
```

Comment:
Should override `isEmpty()` instead of this line, so the required validator will still work.